devtools/dumper-companion.py

```diff
@@ -31,6 +31,7 @@
 from typing import Any
 
 import machfs
+import pycdlib
```

Comment:
Update the documentation at the top of the file. 
Specifically add `pycdlib` to the prerequisites line.